randovania/data/json_data/prime3.txt

```diff
@@ -3812,12 +4678,12 @@ Asset id: 1808913861159744919
 Command Courtyard
 Asset id: 18221971457754325372
 > Dock to Courtyard Passage; Heals? False
-  * Normal Door to Courtyard Passage/Dock to Command Courtyard
+  * Plasma Door to Courtyard Passage/Dock to Command Courtyard
   > Bottom Floor
       Trivial
 
 > Dock to Lift Hub; Heals? False
-  * Normal Door to Lift Hub/Dock to Command Courtyard
+  * Plasma Door to Lift Hub/Dock to Command Courtyard
   > Top Floor
       Trivial
 
```

Comment:
For below:

- Flux Control
  - Change to Morph Ball Door
  - Connect to Lift Hub with requirement "After fan event"
- Missile
  - Connect to Energy Cell as Trivial
- Energy Cell
  - Connect to Fan Event as Trivial
- Fan Event
  - Connect to Lift Hub as Trivial
  - Connect to Flux Control as Trivial
- Barrier Event
  - Connect to Courtyard Passage as Trivial

Reply:
done. I didn't connect the fan event to flux control since we're connecting to the lift hub anyways (which connects to flux)